### ISSUE TITLE
shop generator not working

### ISSUE THREAD

**Issue by mrbme** [Shopify/shopify_app#827]
Hi,

When running the generator I get this response in the terminal

```generate  shopify_app:install --apikey myapikey --secret mysecret
Running via Spring preloader in process 22188
     gemfile  dotenv-rails
      create  config/initializers/shopify_app.rb
      create  config/initializers/session_store.rb
      create  config/initializers/omniauth.rb
      insert  config/initializers/omniauth.rb
      create  app/views/layouts/embedded_app.html.erb
      create  app/views/layouts/_flash_messages.html.erb
      create  app/assets/javascripts/shopify_app.js
      create  app/assets/javascripts/flash_messages.js
      create  config/initializers/user_agent.rb
       route  mount ShopifyApp::Engine, at: '/'
      insert  config/environments/development.rb
    generate  shopify_app:shop_model
/Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie/configuration.rb:97:in `method_missing': undefined method `hosts' for #<Rails::Application::Configuration:0x00007f8356d75128> (NoMethodError)
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:2:in `block in <main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `instance_eval'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `configure'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:1:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:602:in `block (2 levels) in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `block in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `instance_exec'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:228:in `block in tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `tsort_each_child'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:226:in `tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:205:in `tsort_each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:60:in `run_initializers'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/application.rb:361:in `initialize!'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environment.rb:5:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:106:in `preload'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:157:in `serve'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:145:in `block in run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `loop'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
    generate  shopify_app:authenticated_controller
/Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie/configuration.rb:97:in `method_missing': undefined method `hosts' for #<Rails::Application::Configuration:0x00007fb1c9b6a378> (NoMethodError)
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:2:in `block in <main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `instance_eval'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `configure'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:1:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:602:in `block (2 levels) in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `block in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `instance_exec'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:228:in `block in tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `tsort_each_child'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:226:in `tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:205:in `tsort_each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:60:in `run_initializers'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/application.rb:361:in `initialize!'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environment.rb:5:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:106:in `preload'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:157:in `serve'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:145:in `block in run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `loop'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
    generate  shopify_app:home_controller
/Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie/configuration.rb:97:in `method_missing': undefined method `hosts' for #<Rails::Application::Configuration:0x00007f80ffe13638> (NoMethodError)
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:2:in `block in <main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `instance_eval'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/railtie.rb:216:in `configure'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environments/development.rb:1:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:602:in `block (2 levels) in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/engine.rb:601:in `block in <class:Engine>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `instance_exec'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:228:in `block in tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:50:in `tsort_each_child'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:415:in `each_strongly_connected_component_from'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `call'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:347:in `each_strongly_connected_component'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:226:in `tsort_each'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/2.6.0/tsort.rb:205:in `tsort_each'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/initializable.rb:60:in `run_initializers'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/railties-5.2.3/lib/rails/application.rb:361:in `initialize!'
	from /Users/BME/Documents/GitHub/_Rails/shopify_app_2/config/environment.rb:5:in `<main>'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:106:in `preload'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:157:in `serve'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:145:in `block in run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `loop'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application.rb:139:in `run'
	from /Users/BME/.rvm/gems/ruby-2.6.1/gems/spring-2.1.0/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/BME/.rvm/rubies/ruby-2.6.1/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

I've removed my API and Secret keys, but as you can see the generator can't create the model or the controllers. Any insight?

Thanks!

**Comment by tylerball** [comment]
Hello, this should be fixed by #823. Please try upgrading to v11.3.2 and try again.

**Comment by tanema** [comment]
I believe this has been fixed now
